pkg/proto/nbcontract/v1/httpproxyconfig.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+package nbcontract.v1;
+
+message HTTPProxyConfig {
+  // HTTPProxy represents the proxy endpoint/URL using HTTP, that the traffic should go through for making HTTP connections outside the cluster
+  optional string HTTPProxy = 1;
+
+  // HTTPSProxy represents the proxy endpoint using HTTPS, that the traffic should go through for making HTTPS connections outside the cluster
+  optional string HTTPSProxy = 2;
+
+  // NoProxy represents a list of endpoints including IPs, domains, or other network CIDRs that should not be accessed through the proxy
+  repeated string NoProxy = 3;
+
+  // TrustedCA repesents alternative CA certificate content
+  optional string TrustedCA = 4;
```

Comment:
May want to keep "Proxy" somewhere in the name, for consistency with the rest. (The alternative might be to drop "Proxy" everywhere in this context, but seems it would be less readable / clear ...)

Reply:
fair point, let me add Proxy prefix.

Reply:
done.